storage/api/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteStorageMetrics.java

```diff
@@ -0,0 +1,93 @@
+ * place to store them, so that we can verify all of them easily.
+ *
+ * @see kafka.api.MetricsTest
+ */
+public class RemoteStorageMetrics {
+    public static final String REMOTE_LOG_READER_METRICS_NAME_PREFIX = "RemoteLogReader";
+    public static final String REMOTE_LOG_MANAGER_TASKS_AVG_IDLE_PERCENT = "RemoteLogManagerTasksAvgIdlePercent";
+    public static final String TASK_QUEUE_SIZE = "TaskQueueSize";
+    public static final String AVG_IDLE_PERCENT = "AvgIdlePercent";
+    public static final String REMOTE_BYTES_OUT_PER_SEC = "RemoteBytesOutPerSec";
+    public static final String REMOTE_BYTES_IN_PER_SEC = "RemoteBytesInPerSec";
+    public static final String REMOTE_READ_REQUESTS_PER_SEC = "RemoteReadRequestsPerSec";
+    public static final String REMOTE_WRITE_REQUESTS_PER_SEC = "RemoteWriteRequestsPerSec";
+    public static final String FAILED_REMOTE_READ_REQUESTS_PER_SEC = "RemoteReadErrorsPerSec";
+    public static final String FAILED_REMOTE_WRITE_REQUESTS_PER_SEC = "RemoteWriteErrorsPerSec";
+    public static final String REMOTE_LOG_READER_TASK_QUEUE_SIZE = REMOTE_LOG_READER_METRICS_NAME_PREFIX + TASK_QUEUE_SIZE;
+    public static final String REMOTE_LOG_READER_AVG_IDLE_PERCENT = REMOTE_LOG_READER_METRICS_NAME_PREFIX + AVG_IDLE_PERCENT;
+
+    public final static MetricName REMOTE_BYTES_OUT_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", REMOTE_BYTES_OUT_PER_SEC);
+    public final static MetricName REMOTE_BYTES_IN_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", REMOTE_BYTES_IN_PER_SEC);
+    public final static MetricName REMOTE_READ_REQUESTS_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", REMOTE_READ_REQUESTS_PER_SEC);
+    public final static MetricName REMOTE_WRITE_REQUESTS_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", REMOTE_WRITE_REQUESTS_PER_SEC);
+    public final static MetricName FAILED_REMOTE_READ_REQUESTS_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", FAILED_REMOTE_READ_REQUESTS_PER_SEC);
+    public final static MetricName FAILED_REMOTE_WRITE_REQUESTS_PER_SEC_METRIC = getMetricName(
+            "kafka.server", "BrokerTopicMetrics", FAILED_REMOTE_WRITE_REQUESTS_PER_SEC);
+    public final static MetricName REMOTE_LOG_MANAGER_TASKS_AVG_IDLE_PERCENT_METRIC = getMetricName(
+            "kafka.log.remote", "RemoteLogManager", REMOTE_LOG_MANAGER_TASKS_AVG_IDLE_PERCENT);
+    public final static MetricName REMOTE_LOG_READER_TASK_QUEUE_SIZE_METRIC = getMetricName(
+            "org.apache.kafka.storage.internals.log", "RemoteStorageThreadPool", REMOTE_LOG_READER_TASK_QUEUE_SIZE);
+    public final static MetricName REMOTE_LOG_READER_AVG_IDLE_PERCENT_METRIC = getMetricName(
+            "org.apache.kafka.storage.internals.log", "RemoteStorageThreadPool", REMOTE_LOG_READER_AVG_IDLE_PERCENT);
+
+    public static Set<MetricName> allMetrics() {
```

Comment:
we are missing the newly added metrics here.

Reply:
Sorry, I didn't get it. What newly added metrics we missed here? (Too many PRs recently...)

Reply:
the metrics you added in the last commit. e.g. `FAILED_REMOTE_WRITE_REQUESTS_PER_SEC` is missing and hence, our integration tests is not validating that this metric is not emitted when Tiered Storage is turned off.

Reply:
Tests passed now. Thanks.